README.md

```diff
@@ -34,6 +34,6 @@ If you'd like to add your own project, please file a pull request doing so.
 | honeycomb-logger | Nginx proxy that logs to stdout events sent to Honeycomb. | https://github.com/darklang/honeycomb-logger |
 | cl-honeycomb | Common Lisp client for sending events | https://github.com/franzinc/cl-honeycomb/ |
 | terraform-provider-honeycombio | Terraform provider for managing Honeycomb resources | https://github.com/kvrhdn/terraform-provider-honeycombio 
-| honeycomb-create-marker | Github Action for creating Honeycomb Markers | https://github.com/beauknowssoftware/honeycomb-create-marker
-| honeycomb-update-marker | Github Action for updating Honeycomb Markers | https://github.com/beauknowssoftware/honeycomb-update-marker
+| gha-buildevents | Add observability to your GitHub Actions workflows | https://github.com/kvrhdn/gha-buildevents
```

Comment:
```suggestion
| gha-buildevents | Trace GitHub Action workflows and send to Honeycomb | https://github.com/kvrhdn/gha-buildevents
```

Reply:
LGTM. I agree that this is more appropriate 👍🏻